Add efs support (#140)
C2Devel/terraform-provider-croccloud: docs/resources/efs_file_system.md

Reviewer: ```suggestion
### EFS File System with tags
```

Reviewer: оформление ссылки

---
subcategory: "EFS (Elastic File System)"
layout: "aws"
page_title: "aws_efs_file_system"
description: |-
  Creates an Elastic File System (EFS) file system.
---

[default-tags]: https://www.terraform.io/docs/providers/aws/index.html#default_tags-configuration-block
[efs]: http://docs.aws.amazon.com/efs/latest/ug/
[file-system]: https://docs.k2.cloud/ru/services/efs/efs.html#id18

# Resource: aws_efs_file_system

Creates an Elastic File System (EFS) file system.

## Example Usage

### EFS File System with tags

```terraform
resource "aws_efs_file_system" "example" {
  creation_token = "my-product"

  tags = {
    Name = "MyProduct"
  }
}
```

## Argument Reference

The following arguments are supported:

> * `creation_token` - (Optional) A unique name (a maximum of 64 characters are allowed)

Reviewer: лучше одно предложение, одна строка
текст в скобках надо вынести в Constraints (см. оформление выше)

used as reference when creating the Elastic File System to ensure idempotent file
system creation. By default generated by Terraform. See [Elastic File System][efs]
user guide for more information.
* `tags` - (Optional) A map of tags to assign to the file system. If a provider [`default_tags` configuration block][default-tags] is used, tags with matching keys will overwrite those defined at the provider level.

## Attributes Reference

In addition to all arguments above, the following attributes are exported:

* `id` - The ID that identifies the file system.
    * _Example:_  `fs-ccfc0d65`
* `number_of_mount_targets` - The current number of mount targets that the file system has.
* `owner_id` - The account that was used to create the file system. If the file system was created by an IAM user, the parent account to which the user belongs is the owner.

> * `performance_mode` - The file system performance mode. Always `"generalPurpose" for now.

Reviewer: ```suggestion
* `performance_mode` - The file system performance mode. 
    * _Valid values:_ `"generalPurpose"`
```

* `size_in_bytes` - The latest known metered size (in bytes) of data stored in the file system.
   The value is not the exact size that the file system was at any point in time.

>    The structure of this block is [described below](#fixed_response).(#size_in_bytes).

Reviewer: а тут правильная ссылка?
```suggestion
   The structure of this block is [described below](#size_in_bytes).
```

* `tags_all` - A map of tags assigned to the resource, including those inherited from the provider [`default_tags` configuration block][default-tags].

### size_in_bytes

* `value` - The latest known metered size (in bytes) of data stored in the file system.

## Import

> The EFS file systems can be imported using the `id`, e.g.,

Reviewer: ```suggestion
The EFS file systems can be imported using the `id`, for example:
```

---
subcategory: "EFS (Elastic File System)"
layout: "aws"
page_title: "aws_efs_file_system"
description: |-

>   Provides an Elastic File System (EFS) File System resource.

Reviewer: ```suggestion
  Creates an Elastic File System (EFS) file system.
```

---

> [file-system]: https://docs.k2.cloud/ru/services/efs/efs.html#id18

Reviewer: ```suggestion
[default-tags]: https://www.terraform.io/docs/providers/aws/index.html#default_tags-configuration-block
[efs]: (http://docs.aws.amazon.com/efs/latest/ug/
[file-system]: https://docs.k2.cloud/ru/services/efs/efs.html#id18
```

# Resource: aws_efs_file_system

> Provides an Elastic File System (EFS) File System resource.

Reviewer: ```suggestion
Creates an Elastic File System (EFS) file system.
```

}
```

## Argument Reference

The following arguments are supported:

* `creation_token` - (Optional) A unique name (a maximum of 64 characters are allowed)
used as reference when creating the Elastic File System to ensure idempotent file

> system creation. By default generated by Terraform. See [Elastic File System](http://docs.aws.amazon.com/efs/latest/ug/)

Reviewer: ```suggestion
system creation. By default generated by Terraform. See [Elastic File System][efs]
```

user guide for more information.

> * `tags` - (Optional) A map of tags to assign to the file system. If configured with a provider [`default_tags` configuration block](https://www.terraform.io/docs/providers/aws/index.html#default_tags-configuration-block) present, tags with matching keys will overwrite those defined at the provider-level.

Reviewer: ```suggestion
* `tags` - (Optional) A map of tags to assign to the file system. If a provider [`default_tags` configuration block][default-tags] is used, tags with matching keys will overwrite those defined at the provider level.
```

## Attributes Reference

In addition to all arguments above, the following attributes are exported:

> * `availability_zone_id` - The identifier of the Availability Zone in which the file system's One Zone storage classes exist.

Reviewer: см. аналог. коммент

Author: удалил тоже, у нас нет привязки fs к az

> * `id` - The ID that identifies the file system (e.g., fs-ccfc0d65).

Reviewer: ```suggestion
* `id` - The ID that identifies the file system.
    * _Example:_  `fs-ccfc0d65`
```

> * `owner_id` - The AWS account that created the file system. If the file system was createdby an IAM user, the parent account to which the user belongs is the owner.

Reviewer: ```suggestion
* `owner_id` - The account that was used to create the file system. If the file system was created by an IAM user, the parent account to which the user belongs is the owner.
```

* `number_of_mount_targets` - The current number of mount targets that the file system has.

> * `size_in_bytes` - The latest known metered size (in bytes) of data stored in the file system, the value is not the exact size that the file system was at any point in time. See [Size In Bytes](#size-in-bytes).

Reviewer: ```suggestion
* `size_in_bytes` - The latest known metered size (in bytes) of data stored in the file system.
   The value is not the exact size that the file system was at any point in time. 
   The structure of this block is [described below](#fixed_response).(#size_in_bytes).
```

> ### Size In Bytes

Reviewer: название как у атрибута
```suggestion
### size_in_bytes
```
